imap/jmap_ical.c

```diff
@@ -2561,8 +2569,10 @@ participants_from_ical(icalcomponent *comp, json_t *linksbyparticipant)
         json_decref(participants);
         participants = json_null();
     }
+
     free_hash_table(&attendee_by_uri, NULL);
     free_hash_table(&id_by_uri, free);
+    free(schedule_updated);
```

Comment:
Do we need to free schedule_comment as well?

Reply:
right. fixed.